### PR TITLE
⚒️  Fix : Sepolia Issue Review

### PR DESCRIPTION
**Pull request by sk-enya**
:clipboard:  
closes: https://github.com/bobanetwork/gateway/issues/392
closes: https://github.com/bobanetwork/gateway/issues/404

## Overview

- Close the modal and show the appropriate error to the user on failed tx instead of the claim button
- Pending withdrawal exits are missing. Those transactions I submitted the proof don't show up in the history tab.
- Deposit transactions are missing.

### CODE REVIEW

**Review by alvaro-ricotta — APPROVED** [review]

**Review by souradeep-das — APPROVED** [review]

File: src/containers/modals/MultiStepWithdrawalModal/VerticalStepper.tsx [hunk 1]
```diff
@@ -91,7 +92,8 @@ export const VerticalStepper = (props: IVerticalStepperProps) => {
         setLatestLogs(res)
       })
       .catch((error) => {
-        console.log(error)
+        dispatch(openError(`Failed to prove withdrwal!`))
```
Comment (wsdt) [inline src/containers/modals/MultiStepWithdrawalModal/VerticalStepper.tsx:95]:
```suggestion
        dispatch(openError(`Failed to prove withdrawal!`))
```

File: src/containers/modals/MultiStepWithdrawalModal/VerticalStepper.tsx [hunk 2]
```diff
@@ -80,7 +80,8 @@ export const VerticalStepper = (props: IVerticalStepperProps) => {
         })
       })
       .catch((err) => {
-        console.error('ERROR: ', err)
+        dispatch(openError(`Failed to initiate withdrwal!`))
```
Comment (wsdt) [inline src/containers/modals/MultiStepWithdrawalModal/VerticalStepper.tsx:83]:
```suggestion
        dispatch(openError(`Failed to initiate withdrawal!`))
```

File: src/services/transaction.service.ts [hunk 3]
```diff
@@ -203,16 +203,18 @@ class TransactionService {
...
-      (arrVal, othVal) => arrVal.transactionHash === othVal.transactionHash
-    )
+    // console.log(`filteredResults`, filteredResults)
+    // filteredResults = uniqWith(
+    //   filteredResults,
+    //   (arrVal, othVal) => arrVal.transactionHash === othVal.transactionHash
+    // )
 
+    // console.log(`filteredResults unique`, filteredResults)
     return filteredResults.filter((transaction) => transaction?.hash)
```
Comment (wsdt) [inline src/services/transaction.service.ts:218]: Does this remove duplicates? The commented snippet was the fix to Jason's bug.

**Review by wsdt — CHANGES_REQUESTED** [review]
left comments

File: src/services/transaction.service.ts (hunk 3 above)
Comment (sk-enya) [reply src/services/transaction.service.ts:218]: But the comment to fix the Boyuan's bug? LOL

Comment (sk-enya) [reply src/services/transaction.service.ts:218]: You mean https://github.com/bobanetwork/gateway/issues/382 ?

Comment (sk-enya) [reply src/services/transaction.service.ts:211]: Please review the latest commit added fix for the duplication!

Comment (sk-enya) [reply src/services/transaction.service.ts:211]: Fix has been updated in latest commit!

**Review by wsdt — APPROVED** [review]
awesome looks good now!

**Pull request merged**